app/page-partials/my-dashboard/ServiceAccountRoles.tsx

```diff
@@ -3,7 +3,8 @@ import styled from 'styled-components';
 import Select, { MultiValue, ActionMeta } from 'react-select';
 import startCase from 'lodash.startcase';
 import throttle from 'lodash.throttle';
-import { Table, Tabs, Tab, Alert, LastSavedMessage } from '@bcgov-sso/common-react-components';
+import { Tabs, Tab, Alert, LastSavedMessage } from '@bcgov-sso/common-react-components';
+import Table from 'components/TableNew';
```

Comment:
table name

Reply:
consistent with `import Table from 'components/TableNew';` in all import table stmt